docs/en/observability/monitor-nginx-ml.asciidoc

```diff
@@ -0,0 +1,70 @@
+[discrete]
+[[monitor-nginx-ml]]
+== Part 5: Find anomalies in your nginx access logs
+
+Use the nginx machine learning (ML) module from the {integrations-docs}/nginx[Nginx Elastic integration] provides built in ML jobs to help find unusual activity in your nginx access logs.
+Monitoring anomalies in your access logs helps you detect:
+
+* security threats
+* network issues
+* system performance issues
+* operational efficiency issues
+
+[discrete]
+[[monitor-nginx-ml-jobs]]
+=== nginx anomaly detection jobs
+
+The nginx ML module provides the following jobs for detecting anomalies in your access logs:
+
+[[horizontal]]
+Low request rates (low_request_rate_nginx):: Uses the {ml-docs}/ml-count-functions.html#ml-count[`low_count`] function to detect abnormally low request rates. Abnormally low request rates might indicate network issues or other issues are preventing requests from reaching the server.
+Unusual source IPs - high request rates (`source_ip_request_rate_nginx`):: Uses the {ml-docs}/ml-count-functions.html#ml-count[`hight_count`] function to detect abnormally high request rates from individual IP addresses. Many requests from a single IP or small group of IPs might indicate something malicious like a DDoS attack where many requests are sent to overwhelm the server and make it unavailable to users.
```

Comment:
My brain would prefer "a large number of requests" over "many request" but I can't explain why. I guess because "many" somehow seems less precise? Feel free to ignore if you don't see what I mean.